pkg/services/sqlstore/stats_test.go

```diff
@@ -109,4 +116,11 @@ func populateDB(t *testing.T) {
 	}
 	err = AddOrgUser(cmd)
 	assert.Nil(t, err)
+
+	// update 1st user last seen at
+	updateUserLastSeenAtCmd := &models.UpdateUserLastSeenAtCommand{
+		UserId: users[0].Id,
+	}
+	err = UpdateUserLastSeenAt(updateUserLastSeenAtCmd)
+	assert.Nil(t, err)
```

Comment:
We should use `require` instead of `assert` in this function, since if an `assert` call fails, the test will continue instead of aborting immediately (as `require` calls do). It's just chaotic to continue the test after `populateDB` fails. 
```suggestion
	require.NotError(t, err)
```